scripts/report_downstream_test_diffs.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import json
+import os
+import re
+import subprocess
+import typing
+from collections import namedtuple
+
+from playwright.path_utils import get_file_dirname
+
+_dirname = get_file_dirname()
+
+TestCase = namedtuple("TestCase", ["api", "file", "test"])
+
+
+def pytest_test_cases() -> typing.Generator[TestCase, None, None]:
+    p = subprocess.check_call(
+        ["pytest", "--browser", "chromium", "--collect-only", "-q"],
+        cwd=_dirname / ".." / "tests",
+        stdout=subprocess.PIPE,
+    )
+    regex = (
+        r"tests/(?P<api>a?sync)/test_(?P<file>.*)\.py::test_(?P<test>.*)\[chromium\]"
+    )
+    matches = re.finditer(regex, p.stdout.decode(), re.MULTILINE)
+    for match in matches:
+        yield TestCase(
+            match.group("api"), match.group("file"), match.group("test"),
+        )
+
+
+def jest_test_cases(playwright_js_path: str) -> typing.Generator[TestCase, None, None]:
+    p = subprocess.run(
```

Comment:
same here

Reply:
Resolved in latest change

Reply:
It is still .run() instead of check_call()

Reply:
🤦 now it's fixed! 🎉  02af14a